Products/ZenCollector/scheduler.py

```diff
@@ -189,10 +189,8 @@ def __call__(self):
         if self.task.state is TaskStates.STATE_PAUSED and not self.paused:
             self.task.state = TaskStates.STATE_IDLE
 
-        if self.task.state is TaskStates.STATE_IDLE:
-            if self.paused:
-                self.task.state = TaskStates.STATE_PAUSED
```

Comment:
It seems this might be the only place the TaskStates.STATE_PAUSED was ever assigned.  Can you please verify for me that the `PAUSED` state is set when there is a ping down event for collectors other than zencommand?

You can run `zenperfsnmp stats` to get the output of the tasks in zenperfsnmp.